demo/app/android/app/src/main/kotlin/walletsdk/flutter/app/MainActivity.kt

```diff
@@ -195,7 +189,10 @@ class MainActivity : FlutterActivity() {
         val openID4CI = this.openID4CI
             ?: throw java.lang.Exception("openID4CI not initiated. Call authorize before this.")
 
-        return openID4CI.requestCredential(otp)
+        val didVerificationMethod = this.didVerificationMethod
+                ?: throw java.lang.Exception("DID should be created first")
+
+        return openID4CI.requestCredential(otp, didVerificationMethod)
```

Comment:
API surface is changed with this PR. Can we update the documentation as well ?

Reply:
Also, can we add Breaking changes tag to the commit message ?

Reply:
yes, sure

Reply:
done